src/main.ts

```diff
@@ -83,7 +84,7 @@ async function startVue () {
   new Vue({
     vuetify: getVuetify(),
     router: getVueRouter(),
-    store,
+    store: store,
```

Comment:
Old code was OK.

Does this test through to payment OK? I tried using `getVuexStore` but it broken something due to the dynamic store modules.